packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -4,6 +4,7 @@
   --terra-consumer-body-background: url('https://wallpapercave.com/wp/5ynt7Gb.jpg');
   --terra-consumer-nav-active-item-background: var(--terra-consumer-body-background);
   --terra-consumer-nav-item-color: #fff;
+  --terra-base-color: #FFF;
```

Comment:
what is this for?

Reply:
the text color was all messed up and hard to read, i need to remove that prop before merging